src/const.js

```diff
@@ -18,9 +18,9 @@ CONST.HOVER_PSEUDO_CLASS_ATTR = 'data-hammerhead-hovered';
 
 CONST.DOCUMENT_CHARSET = 'hammerhead|document-charset';
 
-CONST.XHR_CORS_SUPPORTED_FLAG   = 0x10;
-CONST.XHR_REQUEST_MARKER_HEADER = 'hammerhead|xhr-request-marker-header';
-CONST.XHR_WITH_CREDENTIALS_FLAG = 0x01;
+CONST.XHR_REQUEST_MARKER_HEADER   = 'hammerhead|xhr-request-marker-header';
+CONST.XHR_CORS_SUPPORTED_HEADER   = 'hammerhead|xhr-cors-supported-header';
+CONST.XHR_WITH_CREDENTIALS_HEADER = 'hammerhead|xhr-with-credentials-header';
```

Comment:
We can definitely move it from here as a part of #68. I suggest creating new dir `src/pipeline/xhr` and move there `same-origin-policy.js` and this constants in `headers.js`.